x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -108,11 +100,13 @@ function useServicesMainStatisticsFetcher() {
       pageSize,
       sortField,
       sortDirection,
+      dataSourceOptions?.bucketSizeInSeconds,
+      dataSourceOptions?.source.documentType,
+      dataSourceOptions?.source.rollupInterval,
```

Comment:
`bucketSizeInSeconds` is a dependency but not used in the hook. Is that intentional? Would it better to add all values of `dataSourceOptions` as dependencies? 

```suggestion
      ...Object.values(dataSourceOptions)
```

Reply:
ah yeah, I can remove bucketSizeInSeconds here.